index.test.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const {
+  addTodo,
+  deleteTodo,
+} = require('./src/mock-add-remove/add-remove-mock.js');
+
+const todos = [
+  {
+    description: "Understanding API's",
+    completed: false,
+    index: '1',
+  },
+];
+
+describe('Test add and delete tasks', () => {
+  test('Add a todo', () => {
+    addTodo();
+    expect(todos.length).toEqual(1);
+  });
+
+  test('Delete a task todo', () => {
+    deleteTodo();
+    expect(todos.length).toBe(0);
+  });
+});
```

Comment:
- [ Required ] Good job adding these tests to your project, however, when I ran the tests the `deleteTodo` test did not pass. It would be better if all your tests pass, please see the screenshot below and rectify this.

## Failing tests
![Screenshot from 2023-03-02 22-34-26](https://user-images.githubusercontent.com/16392046/222534125-83bb6ac9-22b4-4363-a727-6517b3806847.png)